README.md

```diff
@@ -133,6 +144,40 @@ In the case that the request parameters lead to a page outside the result set an
 ### Error Code
 400 Bad Request. If an unknown sort criteria is requested or a not valid ordering keyword is specified
 
+## Design Principles
+In the creation of the REST API, we've tried to follow a few specific design principles listed below
+
+### On the Naming of Endpoints
+Names should be descriptive but reasonably short. Use nouns instead of verbs. Form compounds by concatenating words, all lower case, without punctuation.  For example: `metadatafields`, not `metadata-fields` or `MetadataFields`.
+
+### HATEOAS & HAL
+The REST API supports the [HATEOAS paradigm](https://restfulapi.net/hateoas/) and adopt the [HAL format](http://stateless.co/hal_specification.html) to express links and embedded resources. Links are always expected to be **absolute** to make easier the implementation of "follow link" methods on the REST client side.
+
+Because the API responds using the HAL Format, we distribute it with an [embedded HAL Browser provided by Spring](https://docs.spring.io/spring-data/rest/docs/current/reference/html/#_the_hal_browser).
+
+### Statelessness
+The REST API is [stateless](https://restfulapi.net/statelessness/), meaning the client is responsible for sending any state information to the server when it is needed (there is no session kept on the server between requests).
+
+#### JSON Web Tokens
+[JSON Web Tokens (JWT)](https://jwt.io/) are used to retain state (and authentication) information between requests.
+
+### ALPS - Application Level Profile Semantics
+While not yet implemented, we expect future support for the ALPS metadata (<http://alps.io/>), so a profile link MUST exists from the root of API.
+A profile link as defined in [RFC 6906](<https://tools.ietf.org/html/rfc6906>), is a place to include application level details. The ALPS draft spec (http://tools.ietf.org/html/draft-amundsen-richardson-foster-alps-00)
```

Comment:
"SEE the ALPS draft spec"?  This line looks like something was accidentally left off.

Reply:
Yes, you are correct. The above text already exists in the current README.  I need to further correct it as it doesn't read properly.

Reply:
Cleaned up by just adding "See the ALPS draft spec" for now.  We'll want to clean this section up more later when we actually have *implemented* ALPS